src/Header/MultipleHeaderInterface.php

```diff
@@ -4,5 +4,11 @@
 
 interface MultipleHeaderInterface extends HeaderInterface
 {
+    /**
+     * Convert multiple headers to string representation
+     *
+     * @param array $headers Array of header instances
+     * @return string
+     */
```

Comment:
Please revert, there are no native types on this method, so you are deleting useful information